### PR TITLE
Automated ceph health alert related test

### PR DESCRIPTION
**Pull request by sunilangadi2**
Signed-off-by: Sunil Angadi <sangadi@redhat.com>

# Description
Added new module as alert.py In this we can add more tests related to ceph alerts
Fixes- RHCEPHQE-2779
Polarion- CEPH-83573240

Test Result:
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-SJ73IJ/

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/rados/alerts.py [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    log.info(run.__doc__)
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    # List of alerts to be tested
+    alert_list = ["HEALTH_ERR", "HEALTH_WARN"]
+    for alert in alert_list:
+        if alert == "HEALTH_ERR":
+            if not generate_health_error_alert(alert, cephadm):
+                return 1
+    return 0
```
Comment (pdhiran) [inline tests/rados/alerts.py:28]: Please revert the config changes made during the test.

File: suites/pacific/rados/tier-2_rados_basic_regression.yaml [hunk 2]
```diff
@@ -347,6 +347,12 @@ tests:
           sleep_interval: 60
       desc: Ceph balancer plugins CLI validation in upmap mode
 
+  - test:
+      name: ceph health alerts test
+      polarion-id: CEPH-83573240
```
Comment (pdhiran) [inline suites/pacific/rados/tier-2_rados_basic_regression.yaml:352]: This Test case speaks about verification of alerts on dashboard. The code generates alerts and verifies the same in the ceph status in the cephadm shell. Is this the correct test ID?

File: tests/rados/alerts.py (hunk 1 above)
Comment (sunilangadi2) [reply tests/rados/alerts.py:28]: @pdhiran Yes done that in that call itself refer line no 70 I am resetting default config.

File: suites/pacific/rados/tier-2_rados_basic_regression.yaml (hunk 2 above)
Comment (sunilangadi2) [reply suites/pacific/rados/tier-2_rados_basic_regression.yaml:352]: Yes dashboard alerts are coming from ceph health status hence I added that ID,
If you know any other tests in polarion let me know we can mark them also automated
also can we use the same polarion ID for other alerts to be automated

**Comment by sunilangadi2** [comment]
As discussed with @psathyan since alerts are generic to overall ceph and also we have alerts for OSD, MON, MGR, MDS, RGW so will create a separate folder for alerts instead of combining it in RADOS and suite to test all alerts and separate polarion for each alert
Hence will close this PR and raise a new one soon.

**Pull request closed without merging**